net/advanced-operations/working-with-javascript/_index.md

```diff
@@ -106,6 +106,75 @@ A new property named JavaScript is added in Document class which has JavaScript
 
 {{< gist "aspose-pdf" "7e1330795d76012fcb04248bb81d45b3" "Examples-CSharp-AsposePDF-Working-Document-AddRemoveJavascriptToDoc-AddRemoveJavascriptToDoc.cs" >}}
 
+### Setting Expiry Date of a PDF Document Using JavaScript Actions
+
+Aspose.PDF allows you to set an expiry date for a PDF document by embedding JavaScript Actions. This functionality ensures that the PDF becomes inaccessible after a specified date and time, enhancing document security and control. By leveraging JavaScript Actions, you can define precise expiration conditions down to the second, ensuring that the document's accessibility is tightly regulated.
+
+**You can achieve this by following these steps**
+
+1. **Initialize Document:** Create a new PDF document and add a blank page or open an existing PDF document.
+2. **Define Expiry Date and Time:** Set the date and time after which the document will expire.
+3. **Prepare JavaScript Code:** 
+    - Retrieve the current date and time.
+    - Define the exact expiry date and time, considering that months are zero-based in JavaScript.
+    - Compare the current date and time with the expiry date and time.
+    - If the current date and time exceed the expiry date and time, display an alert and close the document.
+4. **Set Open Action:** Associate the JavaScript action with the document's open action.
+5. **Save Document:** Save the PDF with the embedded JavaScript that enforces the expiry condition.
+
+Below you can find code snippets demonstrating this functionality in both C# (.NET) and Java.
```

Comment:
Below are code snippets demonstrating this functionality in both C# (.NET) and Java.